src/html/blocks/listitem.js

```diff
@@ -8,11 +8,20 @@ const serialize = Serializer()
     .matchType(BLOCKS.LIST_ITEM)
     .then(state => {
         const node = state.peek();
-        const inner = state.serialize(node.nodes);
+        let inner = state.serialize(node.nodes);
+        let tag = '<li>';
+
+        const isTaskList = node.data.has('checked');
+        const isChecked = node.data.get('checked');
+
+        if (isTaskList) {
+            tag = '<li class="task-list-item">';
```

Comment:
Could you rather put `className` as a variable

`className = ' class="task-list-item"'`
or 
`className = ''`